assets/extensions/login-template.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Login</title>
+  </head>
+  <body>
+
+    {{ if .Error }}
+      <div>{{ .Error }}</div>
+    {{ end }}
+
+    <form action="{{ .Action }}" method="POST">
+      <input type="hidden" name="then" value="{{ .Values.Then }}">
+      <input type="hidden" name="csrf" value="{{ .Values.CSRF }}">
+
+      <label for="inputUsername">Username</label>
+      <input type="text" id="inputUsername" placeholder="" tabindex="1" autofocus="autofocus" type="text" name="username" value="{{ .Values.Username }}">
+
+      <label for="inputPassword">Password</label>
+      <input type="password" id="inputPassword" placeholder="" tabindex="2" type="password" name="password" value="">
+
+      <button type="submit" tabindex="4">Log In</button>
+
+    </form>
+
+  </body>
+</html>
```

Comment:
we'll want it in a go constant anyway for testing (we'll want to test a locked version of the first example template, the current example template, and the default template), so a command to output it wouldn't bother me